src/translator_functions.php

```diff
@@ -9,17 +9,19 @@
  *
  * @return string
  */
-function __($original)
+function __($original, $args = null)
 {
     $text = BaseTranslator::$current->gettext($original);
 
-    if (func_num_args() === 1) {
+    if ( ! $args ) {
```

Comment:
I agree: the previous code is much more safe: we may have cases when we pass `null` too, and that should be passes to the vsprintf below